.github/workflows/merge-to-stage.js

```diff
@@ -9,6 +9,8 @@ const {
 const PR_TITLE = '[Release] Stage to Main';
 const SEEN = {};
 const REQUIRED_APPROVALS = process.env.REQUIRED_APPROVALS || 2;
+const MAX_MERGES = process.env.MAX_PRS_PER_BATCH || 8;
```

Comment:
I don't quite remember how we landed on `8`. How do you suggest we update the value should we need to adapt it?

Reply:
We did a reasonable estimate based on the average of PRs we see +2

To adapt the value, we can add a [secret](https://github.com/adobecom/milo/settings/secrets/actions) and can just update that, rather than having to make a PR